doc/model.rst

```diff
@@ -0,0 +1,719 @@
+Client:
+
+.. code-block:: python
+
+    from smartredis import Client
+    from smartredis import *
+    import numpy as np
+
+    # Initialize a Client
+    client = Client(cluster=False)
+
+SmartRedis offers the function, ``Client.set_data_source()`` that
+will add a prefix to the key name when using ``Client.get_tensor()``.
+
+.. code-block:: python
+
+    client.set_data_source("producer_1")
+    # Searching for key name: producer_1.tensor
+    client.get_tensor("tensor")
+    client.set_data_source("producer_2")
+    # Searching for key name: producer_2.tensor
+    client.get_tensor("tensor")
+
+Next, output the tensor values to validate correctness:
+
+.. code-block:: python
+
+    client.log_data(LLInfo, f"1: {val1}")
+    client.log_data(LLInfo, f"2: {val2}")
+
+The Experiment Driver script
+============================
+Begin by initializing a SmartSim Experiment object
+with the required import modules and launching
+a standalone database:
+
+.. code-block:: python
+
+    from smartsim import Experiment
+    from smartsim.log import get_logger
+
+    logger = get_logger("Experiment Log")
+    # Initialize the Experiment
+    exp = Experiment("getting-started", launcher="auto")
+
+    # Initialize a single sharded database
+    single_shard_db = exp.create_database(port=6379, db_nodes=1, interface="ib0")
+    exp.generate(single_shard_db, overwrite=True)
+    exp.start(single_shard_db)
+
+Next, lets create each Model with runsettings and key prefixing
+beginning with model 1: `producer_1.py`. Initialize the run
+settings for the Model, then create the Model:
+
+.. code-block:: python
+
+    # Initialize a RunSettings object
+    producer_settings_1 = exp.create_run_settings(exe=exe_ex, exe_args="/path/to/producer_1.py")
+    producer_settings_1.set_nodes(1)
+    producer_settings_1.set_tasks_per_node(1)
+    producer_1 = exp.create_model("producer_1", producer_settings_1)
+
+Since this model will be producing and sending tensors to the standard Orchestrator,
+we need to enable key prefixing like below:
+
+.. code-block:: python
+
+    producer_1.enable_key_prefixing()
+
+Now repeat this process for the second producer model:
+.. code-block:: python
+
+    # Initialize a RunSettings object
+    producer_settings_2 = exp.create_run_settings(exe=exe_ex, exe_args="/path/to/producer_2.py")
+    producer_settings_2.set_nodes(1)
+    producer_settings_2.set_tasks_per_node(1)
+    producer_2 = exp.create_model("producer_2", producer_settings_2)
+    producer_2.enable_key_prefixing()
+
+Next, lets create the consumer model that will be request the
+keys from `producer_1` and `producer_2`:
+
+.. code-block:: python
+
+    # Initialize a RunSettings object
+    consumer_settings = exp.create_run_settings(exe=exe_ex, exe_args="/path/to/consumer.py")
+    consumer_settings.set_nodes(1)
+    consumer_settings.set_tasks_per_node(1)
+    consumer = exp.create_model("consumer", consumer_settings)
```

Comment:
Don't we need to register incoming entities on the consumer? Let's run these scripts and make sure they work.